EduVPN/Services/PersistenceService.swift

```diff
@@ -318,6 +322,22 @@ extension PersistenceService {
             }
         }
 
+        var wireGuardPrivateKey: Data? {
+            get {
+                if let data = try? Data(contentsOf: wireGuardPrivateKeyURL),
+                    let clearTextData = Crypto.shared.decrypt(data: data) {
+                    return clearTextData
+                }
+                return nil
+            }
+            set(value) {
+                let data = value ?? Data()
+                if let encryptedData = try? Crypto.shared.encrypt(data: data) {
```

Comment:
`if let data = value, let encrypted...` Can't see why using Data() would be useful ever.

Reply:
That code is just for consistency. If we ever set `wireGuardPrivateKey` to `nil`, the data on disk should get deleted.